test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated.test.cpp

```diff
@@ -416,8 +416,10 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Models_Propagated, Calcul
     const Composite satelliteGeometry(Cuboid(
         {0.0, 0.0, 0.0}, {Vector3d {1.0, 0.0, 0.0}, Vector3d {0.0, 1.0, 0.0}, Vector3d {0.0, 0.0, 1.0}}, {1.0, 2.0, 3.0}
     ));
+
+    // Drag coefficient to 0.0 to neglect atmospheric drag
```

Comment:
This is no longer required yeah? Since we updated the model.

Reply:
Honestly though you can just leave it my dynamics MR revamps all this. I'm going to need some help from you with the rebase :P Maybe we can pair on that.

Reply:
Yeah that's technically true. Although I honestly don't mind keeping it there to be explicit in the test that we're not considering drag 🤷

Reply:
💯 let's crank through the rebase together